### PR TITLE
loading JSON translations

### PR DESCRIPTION
**Pull request by Eric-all**

### CODE REVIEW

**Comment by Eric-all** [comment]
@nWidart lock https://github.com/nWidart/laravel-modules/issues/400

**Comment by Eric-all** [comment]
@nWidart  look , https://github.com/laravel/framework/pull/20599   😃

**Comment by nWidart** [comment]
Thanks, though this seems to be adding I/O operations. Don't you think it's easy for someone who wants json translations to add it?

Maybe it could be documented on the docs.

File: src/Module.php [hunk 1]
```diff
@@ -190,6 +190,7 @@ protected function registerTranslation()
 
         if (is_dir($langPath)) {
             $this->loadTranslationsFrom($langPath, $lowerName);
+            $this->loadJSONTranslationsFrom($langPath);
```
Comment (mikemand) [inline src/Module.php]: This should be `loadJsonTranslationsFrom`. JSON is not capitalized: [Illuminate/Support/ServiceProvider.php#L107-L116](https://github.com/laravel/framework/blob/dfcb8e372e7efb0e95c5844571c0701e91b7f2e6/src/Illuminate/Support/ServiceProvider.php#L107-L116)

**Comment by nWidart** [comment]
Thank you for the contribution, but this is something that's easily added manually and avoids extra load.

**Pull request closed without merging**